index.html

```diff
@@ -1,1227 +1,1530 @@
+									media="(min-width:1920px)">
+								<source srcset="img/ourVisitors/x1000/container_3/photo33_x1000.jpg" type="image/jpeg"
+									media="(min-width:1000px)">
+								<source srcset="img/ourVisitors/x768/container_3/photo33_x768.jpg" type="image/jpeg"
+									media="(min-width:768px)">
+								<img class="photo" src="img/ourVisitors/x320/container_3/photo33_x320.jpg" alt="photo">
+							</picture>
+						</div>
+						<div class="photo__item">
+							<picture>
+								<source srcset="img/ourVisitors/x1920/container_3/photo34_x1920.jpg" type="image/jpeg"
+									media="(min-width:1920px)">
+								<!-- <source srcset="img/ourVisitors/x1000/container_3/photo34_x1000.jpg" type="image/jpeg" media="(min-width:1000px)"> -->
+								<!-- <source srcset="img/ourVisitors/x768/container_3/photo34_x768.jpg" type="image/jpeg" media="(min-width:768px)"> -->
+								<img class="photo" src="img/ourVisitors/x320/container_3/photo34_x320.jpg" alt="photo">
+							</picture>
+						</div>
+						<div class="photo__item">
+							<picture>
+								<!-- <source srcset="img/ourVisitors/x1920/container_3/photo35_x1920.jpg" type="image/jpeg" media="(min-width:1920px)"> -->
+								<!-- <source srcset="img/ourVisitors/x1000/container_3/photo35_x1000.jpg" type="image/jpeg" media="(min-width:1000px)"> -->
+								<!-- <source srcset="img/ourVisitors/x768/container_3/photo35_x768.jpg" type="image/jpeg" media="(min-width:768px)"> -->
+								<img class="photo" src="img/ourVisitors/x320/container_3/photo35_x320.jpg" alt="photo">
+							</picture>
+						</div>
+						<div class="photo__item">
+							<picture>
+								<!-- <source srcset="img/ourVisitors/x1920/container_3/photo36_x1920.jpg" type="image/jpeg" media="(min-width:1920px)"> -->
+								<!-- <source srcset="img/ourVisitors/x1000/container_3/photo36_x1000.jpg" type="image/jpeg" media="(min-width:1000px)"> -->
+								<!-- <source srcset="img/ourVisitors/x768/container_3/photo36_x768.jpg" type="image/jpeg" media="(min-width:768px)"> -->
+								<img class="photo" src="img/ourVisitors/x320/container_3/photo36_x320.jpg" alt="photo">
+							</picture>
+						</div>
 					</div>
 				</div>
-			</section>
-			<section class="ourVisitors">
-				<div class="ourVisitors__container _container">
-					<div class="ourVisitors__title title_every">
-						<h2>Наши посетители</h2>
+				<div class="ourVisitors__sliderCheckbox sliderCheckbox">
+					<button id="button_prev_ourVisitors" class="slider_button" disabled>
+						<div class="arrow arrow_prev"></div>
+					</button>
+					<div id="slider_range" class="slider_range">
+						<div id="slider_thumb" class="slider_thumb"></div>
 					</div>
-					<div id="ourVisitors__photo_container" class="ourVisitors__photo_container">	
-						<div class="ourVisitors__photo">
-							<div class="photo__item">
+					<button id="button_next_ourVisitors" class="slider_button">
+						<div class="arrow arrow_next"></div>
```

Comment:
google if it is a good idea to use a `<div>` inside a `<button>`

Reply:
arrows are not in the center of the buttons
![image](https://user-images.githubusercontent.com/37267977/220743237-1c399cca-805c-4097-a61f-5a1705711c54.png)

check all the places

Reply:
Done.

Reply:
You still could have used one span with borders for it, you just needed to calculate transform: trranslate right